src/autopas/LogicHandler.h

```diff
@@ -711,6 +742,12 @@ class LogicHandler {
    */
   unsigned int _verletClusterSize;
 
+  /**
+   * Array of int which contains number of steps after which rebuild occurs
+   * This is used to calculate the mean rebuild frequency
+   */
+  std::vector<int> _rebuildDistances;
```

Comment:
How about renaming this to `_rebuildIntervals`? At least for me "distances" is a bit misleading here.

Reply:
makes sense, done :)